server/routes/cart-router.js

```diff
@@ -0,0 +1,13 @@
+const express = require('express');
+const {
+   getCartItems,
+   addCartItem,
+   deleteCartItem,
+} = require('../controllers/cart');
+const router = express.Router();
+
+router.route('/').get(getCartItems).post(addCartItem);
+

```

Comment:
You should write .get(getCartItems) in a new line and .post(addCartItem) in the next one. It's just nitpicking but it easier to read and intuitive

Reply:
LOL that's prettier's doing. I did the same as you mentioned on repl but prettier wants the code to look like this😂

Reply:
prettier doesnt makes it look prettier i guess :rofl: